api/worker.py

```diff
@@ -26,18 +28,23 @@ async def main():
     saver = Saver(database=database)
     retriever = Retriever(database=database)
 
+#    sqsClient = boto3.client(service_name='sqs', region_name='us-east-1', aws_access_key_id=os.environ['AWS_KEY'], aws_secret_access_key=os.environ['AWS_SECRET'])
```

Comment:
would be good to remove this stuff from prod - its good to keep a branch locally with stuff like this where you can commit it so you dont loose it